src/main/java/seedu/multitasky/model/entry/EntryList.java

```diff
@@ -132,12 +132,6 @@ public void updateEntry(ReadOnlyEntry target, ReadOnlyEntry editedEntry)
 
         Entry entryToUpdate = internalList.get(index);
 
-        if (internalList.contains(editedEntry)) {
-            if (!isArchivedOrDeletedFloatingTask(editedEntry)) {
-                throw new DuplicateEntryException();
-            }
-        }
-
```

Comment:
Ah, I think the cases whereby `DuplicateEntryException` should be thrown are actually covered in our existing tests cases, that's why you're failing Travis. Just merge my branch `fix-edit-tag-bug` into your branch and those tests expecting `DuplicateEntryException`s should be fixed.